content/articles/2018-12-14-tidyverse-and-r-lib-year-in-review.Rmarkdown

```diff
@@ -0,0 +1,45 @@
+---
+title: "tidyverse and r-lib: a year in review"
+author: Mara Averick
+date: '2018-12-14'
+slug: tidyverse-and-r-lib-year-in-review
+description: > 
+  A look back on 2018 in the tidyverse and beyond.
+categories:
+  - other
+photo:
+  url: https://unsplash.com/photos/SshYpuf607g
+  author: Aperture Vintage
+---
+
+*Hadley Wickham, Mara Averick, Jenny Bryan, Gábor Csárdi, Romain François, Alex Hayes, Lionel Henry, Jim Hester, Max Kuhn, Thomas Lin Pedersen, Davis Vaughan*
+
+We added significant roster depth this year, with [Davis Vaughan](https://twitter.com/dvaughan32/), [Thomas Lin Pedersen](https://twitter.com/thomasp85/), [Alex Hayes](https://twitter.com/alexpghayes), and [Romain François](https://twitter.com/romain_francois/) all joining the squad.
```

Comment:
Should mention Alex is part time to not startle people

Reply:
Still missing this?

Reply:
Whoops. Sorry, I missed like half these comments somehow.

Reply:
Also, based on a convo with @topepo, I think it might be a good idea to strike Alex from this list…

Reply:
Agreed